LSP-eslint.sublime-settings

```diff
@@ -4,10 +4,18 @@
 			"languageId": "javascript",
 			"document_selector": "source.js",
 		},
+		{
+			"languageId": "javascriptreact",
+			"document_selector": "source.jsx | source.js.react",
+		},
```

Comment:
This should go above the `javascript` languageId because there's a tiny implementation detail bug in our matcher.

Reply:
Ninja-merged. I was too late :)

Reply:
Sorry. :)
Changed directly on master: 70ccd4a7774b0a8609cb9ac175767d0b2937086b

Reply:
Looks like the reasoning from [this PR](https://github.com/sublimelsp/LSP-typescript/pull/17#issue-422255991) applies here as well. Not a bug per se.